File: examples/src/Rendering-Printing/RenderingEmoji.cs
import aspose.slides as slides
using Aspose.slides.Examples.CSharp
using System
using System.Collections.Generic
using System.Linq
using System.text
using System.Threading.Tasks

namespace CSharp.Rendering_Printing
{
	public class RenderingEmoji
	{
		#ExStart:RenderingEmoji
		public static void Run()
		{
			dataDir = RunExamples.GetDataDir_Rendering()

			Presentation pres = new Presentation(dataDir+"input.pptx")

			pres.save(dataDir+"emoji.pdf",Aspose.slides.Export.SaveFormat.Pdf)
         }

	}
      #ExEnd:RenderingEmoji
}

File: examples/src/Rendering-Printing/RenderingEmoji.py
import aspose.slides as slides

def rendering_emoji():
	#ExStart:RenderingEmoji
	dataDir = "./examples/data/"
	outDir = "./examples/out/"

	with slides.Presentation(dataDir + "rendering_emoji.pptx") as pres:
		pres.save(outDir + "rendering_emoji_out.pdf", slides.export.SaveFormat.PDF)
	#ExEnd:RenderingEmoji
